SETUP.md

```diff
@@ -131,4 +131,9 @@ To update and reset the database, you have to bring down compose, delete the vol
 
 ## Restarting psinode
 
-If you restart psinode and you are a block producer, you will need to unlock your hsm device again that holds your block signing keys. This can be done in the `x-admin` app.
\ No newline at end of file
+If you restart psinode and you are a block producer, you will need to unlock your hsm device again that holds your block signing keys. This can be done in the `x-admin` app.
+
+## Adding dynamic DNS
```

Comment:
This should probably specify that it's optional.
And under what conditions should the user of this tool consider adding ddns?

Reply:
Added